INSTALLING.md

```diff
@@ -139,13 +135,14 @@ pip install <wheel_file.whl>
 ```
 
 If you are going to try our pre-built models in `models/`, then you must also
-further install their requirements:
+further install their required dependencies and environment variables:
 
 ```sh
+export PYTHONPATH=$(pwd)
```

Comment:
people might try to build from source first and they may have already exported the env variables relevant to source build which will break the whl build. 

good idea to let user know to undo the env variables set for source build if they are going to try whl build next.